.travis.yml

```diff
@@ -2,6 +2,10 @@ language: python
 sudo: false
 matrix:
   include:
+  - python: "3.7"
+    dist: xenial
```

Comment:
Why's this needed?

Reply:
Last time I checked 3.7 was only available on xenial on travis. That may no longer be the case but its what we do in our other projects as well:
https://github.com/aws/aws-cli/blob/develop/.travis.yml#L23
https://github.com/boto/botocore/blob/develop/.travis.yml#L23
https://github.com/boto/boto3/blob/develop/.travis.yml#L23